packages/nimble-blazor/NimbleBlazor/Components/NimbleMappingIcon.razor.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Microsoft.AspNetCore.Components;
+
+namespace NimbleBlazor;
+
+public partial class NimbleMappingIcon<TKey> : ComponentBase
+{
+    /// <summary>
+    /// Gets or sets the key value being mapped.
+    /// </summary>
+    [Parameter]
+    public TKey? Key { get; set; }
+
+    /// <summary>
+    /// Gets or sets the text used for the tooltip and accessible name of the icon.
+    /// </summary>
+    [Parameter]
+    public string? Text { get; set; }
+
+    /// <summary>
+    /// Gets or sets the (element name of the) Nimble icon being mapped to.
```

Comment:
We don't have a good way to refer to these element names in Blazor, do we? And the name that is required isn't even the Blazor component name that a Blazor developer would be familiar with.

It seems like we either need some sort of docs telling Blazor developers how to reference the correct icon name, or more likely, we need a tech debt item to export icon names as constants in Blazor.

Reply:
Good point. Filed item #1458.